recipes/diagrams/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "diagrams",
+  "name": "Diagrams",
+  "version": "1.1.0",
+  "license": "MIT",
+  "config": {
+    "serviceURL": "https://app.diagrams.net/",
+    "hasCustomUrl": true
```

Comment:
This option is usually only needed when e.g. services offer a hosted option.  

Looking at the application itself and the URL, I don't think we need this config option, wdyt?